packages/eslint-config-twilio/rules/base.js

```diff
@@ -0,0 +1,61 @@
+// These are the rules that prettier will take care of
+module.exports = {
+  'comma-dangle': 'off',
+  'array-bracket-newline': 'off',
+  'array-bracket-spacing': 'off',
+  'array-element-newline': 'off',
+  'arrow-parens': 'off',
+  'arrow-spacing': 'off',
+  'block-spacing': 'off',
+  'brace-style': 'off',
+  'comma-spacing': 'off',
+  'comma-style': 'off',
+  'computed-property-spacing': 'off',
+  'dot-location': 'off',
+  'eol-last': 'off',
+  'func-call-spacing': 'off',
+  'function-paren-newline': 'off',
+  'generator-star-spacing': 'off',
+  'implicit-arrow-linebreak': 'off',
+  'indent': 'off',
+  'jsx-quotes': 'off',
+  'key-spacing': 'off',
+  'keyword-spacing': 'off',
+  'linebreak-style': 'off',
+  'multiline-ternary': 'off',
+  'new-parens': 'off',
+  'newline-per-chained-call': 'off',
+  'no-extra-parens': 'off',
+  'no-floating-decimal': 'off',
+  'no-mixed-spaces-and-tabs': 'off',
+  'no-multi-spaces': 'off',
+  'no-multiple-empty-lines': 'off',
+  'no-whitespace-before-property': 'off',
+  'object-curly-newline': 'off',
+  'object-curly-spacing': 'off',
+  'object-property-newline': 'off',
+  'one-var-declaration-per-line': 'off',
+  'operator-linebreak': 'off',
+  'padded-blocks': 'off',
+  'quote-props': 'off',
+  'rest-spread-spacing': 'off',
+  'semi': 'off',
+  'semi-spacing': 'off',
+  'semi-style': 'off',
+  'space-before-blocks': 'off',
+  'space-before-function-paren': 'off',
+  'space-in-parens': 'off',
+  'space-infix-ops': 'off',
+  'space-unary-ops': 'off',
+  'switch-colon-spacing': 'off',
+  'template-curly-spacing': 'off',
+  'template-tag-spacing': 'off',
+  'unicode-bom': 'off',
+  'wrap-iife': 'off',
+  'yield-star-spacing': 'off',
+  'arrow-body-style': 'off',
+  'max-len': 'off',
```

Comment:
hm, what will this mean for our max-len override?

Reply:
All add it to prettier